src/Spark.NetCore/Startup.cs

```diff
@@ -59,7 +59,7 @@ public void Configure(IApplicationBuilder app, IHostingEnvironment env)
                 app.UseHsts();
             }
             
-            app.UseHttpsRedirection();
+            // app.UseHttpsRedirection();
```

Comment:
The https redirection does not work well when running in Docker. OK to switch off?